yt/chyt/controller/go.mod

```diff
@@ -1,6 +1,6 @@
 module go.ytsaurus.tech/yt/chyt/controller
 
-go 1.20
+go 1.23
```

Comment:
We've fixed the compilation under go 1.20 in the master. Please, rebase the brunch and remove this